types/instance.d.ts

```diff
@@ -66,7 +66,7 @@ declare module 'aedes' {
     brokers: Readonly<Brokers>
 
     constructor(option?: AedesOptions)
-    handle: (stream: Connection, request: IncomingMessage) => Client
+    handle: (stream: Connection, request?: IncomingMessage | null) => Client
```

Comment:
```suggestion
    handle: (stream: Connection, request?: IncomingMessage) => Client
```

Reply:
this change is required based on the changes required in the javascript code.

Reply:
See previous answer